docs/introduction.md

```diff
@@ -1,41 +1,31 @@
 ---
 id: intro
-title: Welcome to IOV 
+title: Welcome to IOV! 
 sidebar_label: Introduction 
 ---
 
-We'll point you to the docs and repos you need to get started: first, this document contains a "How to Get Started" section and further below, we have included a brief introduction to IOV for those not already familiar with the project.
-
 **IOV** is building a Universal Name Service for Wallets and Dapps.
 
-We are providing a IOV Name Service which is a register for names used in the blockchain ecosystem, such as personal value addresses, asset names, blockchain names for developers or companies. We believe naming things makes them easier to use for everyone.
-
-We are also providing a toolkit to build apps that can easily use and swap all assets from all blockchains. We believe the future of apps is there.
```

Comment:
I think this sentence is valuable. It indicates that IOV develops toolkit apart from products. We can keep it

Reply:
I've just added back the paragraph with minor twists.